samples/nodes/src/main/java/com/gluonhq/emoji/samples/Demo.java

```diff
@@ -48,28 +52,35 @@ public class Demo extends Application {
     @Override
     public void start(Stage primaryStage) {
 
-        // Get a single text string with all unicode characters for all emojis
-        String text = getAllEmojisAsUnicodeString();
-
-        // Convert text to Text (if delimiter is not empty) and Image nodes (all emojis)
-        List<Node> emojiNodes = TextUtils.convertToTextAndImageNodes(text);
-        emojiNodes.stream()
-                .filter(ImageView.class::isInstance)
-                .forEach(node -> {
-                    String unified = (String) node.getProperties().get(EmojiImageUtils.IMAGE_VIEW_EMOJI_PROPERTY);
-                    EmojiData.emojiFromCodepoints(unified).ifPresent(emoji ->
-                            Tooltip.install(node, new Tooltip("Emoji: " + emoji.getName() + "\n" + unified)));
-                });
-        // Add nodes to textFlow
+        ProgressIndicator progressIndicator = new ProgressIndicator();
+        Scene scene = new Scene(new StackPane(progressIndicator), 1230, 800);
```

Comment:
I'd only add the progressIndicator in case emojis are not initialised:
```
ScrollPane pane = new ScrollPane();
Scene scene = new Scene(pane, 1230, 800);
if (!EmojiLoaderFactory.getEmojiImageLoader().isInitialized()) {
        ProgressIndicator progressIndicator = new ProgressIndicator();
        pane.setContent(progressIndicator);
        EmojiLoaderFactory.getEmojiImageLoader().initialize().thenAccept(aBoolean -> {
              processEmojis();
              pane.setContent(textFlow);
        });
} else {
        processEmojis();
        pane.setContent(textFlow);
}
```

Reply:
Got it.

Adding it directly to ScrollPane always places the ProgressIndicator in the top left left, where as I was adding to the Scene with StackPane as parent to show it at the center.